server/auth.py

```diff
@@ -45,6 +46,26 @@ def has_access(user_id, target_character_id, self_access=False):
 def load_user(user_id):
     return User.get(user_id)
 
+# Decorator for admin access
```

Comment:
I'd say role requirements are an unimplemented feature you should probably wait and have me implement. I'm fine with this sticking right now, but keep that in mind.

Reply:
ok